awx/ui/client/src/templates/job_templates/edit-job-template/job-template-edit.controller.js

```diff
@@ -316,6 +316,9 @@ export default
                                         msg: 'Call failed. Returned status: ' + status
                                     });
                                 });
+                        } else if ($scope.job_template_obj.summary_fields.user_capabilities.start) {
```

Comment:
I think this should work? @mabashian thoughts?

Reply:
@jakemcdermott I think this will work for users that have the ability to start but not edit.  The field is still going to be missing for users with only read access though (like a system auditor):

<img width="1680" alt="Screen Shot 2020-03-13 at 12 45 13 PM" src="https://user-images.githubusercontent.com/9889020/76641781-830d3700-6528-11ea-922b-8dfc7c63f7ca.png">